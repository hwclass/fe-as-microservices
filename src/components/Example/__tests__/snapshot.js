// snapshot.js
import React from 'react';
import Example from '../';
import renderer from 'react-test-renderer';

test('Example component parses default text', () => {
  const component = renderer.create(
    <Example/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});