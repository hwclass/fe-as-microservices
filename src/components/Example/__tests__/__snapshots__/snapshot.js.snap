// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example component parses default text 1`] = `
<div
  className="style__paragraph--2ELag"
>
  <span>
    Example Component
  </span>
</div>
`;
